flixel/FlxCamera.hx

```diff
@@ -493,8 +493,11 @@ class FlxCamera extends FlxBasic
 	var _point:FlxPoint = FlxPoint.get();
 
 	/**
-	 * Internal, the filters array to be applied to the camera.
+	 * The filters array to be applied to the camera. Default value to null for performance reasons.
 	 */
+	public var filters:Array<BitmapFilter>;
```

Comment:
let's make this 
```suggestion
	public var filters:Null<Array<BitmapFilter>> = null;
```
might as well make _filters Null<Array<BitmapFilter>> as well